Dependencies/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# cmake 3.18+ to have the ARCHIVE_EXTRACT sub-command for files
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Should we be consistent in what version of CMake we require? I.e. should we change this everywhere instead of just this file?

Reply:
My reasoning here was that this is the only file that uses features up to this version, and we already have a handful of different cmake_minimum_required versions across BabylonNative (and then differences between our submodules), so we have a better idea of what parts of the cmake code need which version. That being said it looks like the root CMakeLists has the maximum minimum_required version across all of these files, so it makes sense to me to need to update that version as well.